mlir/lib/Conversion/AIRToAIEPass.cpp

```diff
@@ -2579,12 +2579,12 @@ class AIRToAIEPass : public AIRToAIEBase<AIRToAIEPass> {
     if (clTestPatterns.find("to-aie-mlir") != std::string::npos) {
       std::vector<std::pair<AIE::DeviceOp, air::HerdOp>> aie_modules;
       std::map<AIE::TileOp, air::HerdOp> tileToHerdMap;
-      AIRToAIEOptions options = {.col_offset = clColOffset,
-                                 .row_offset = clRowOffset,
-                                 .emit_while = clEmitWhileLoop,
-                                 .emit_herd_lock = clEmitHerdLock,
-                                 .generate_shim_dma = clGenerateShimDMA,
-                                 .device = *device};
+      AIRToAIEOptions options = {/*.col_offset = */ clColOffset,
```

Comment:
Just compile with C++20 since this is not up-streamed?

Reply:
Iree isn't compiling with c++20.. just looking for the simple fix here.

Reply:
That is my point. When you compile IREE yourself, just add `-std=c++20`.
For all our SYCL work I had `-std=c++23` because I do not care about being compatible with old-compilers upstream.